pkg/sysinfo/sysinfo.go

```diff
@@ -102,6 +117,128 @@ type cgroupPids struct {
 	PidsLimit bool
 }
 
+// ValidateHugetlb check whether hugetlb pagesize and limit legal
+func (c cgroupHugetlbInfo) ValidateHugetlb(pageSize string, limit uint64) (string, []string, error) {
```

Comment:
Do all these validation functions belong in `pkg/sysinfo`?  They seem to be related to the container configuration, not the system

Reply:
one can have different page sizes and default size through configuration in system cmdline(default_hugepagesz=xxx hugepagesz=xxx .etc), it will be easier for sysinfo to handle this.